tests/auth_tests/test_validators.py

```diff
@@ -127,6 +129,26 @@ def test_validate(self):
             UserAttributeSimilarityValidator(user_attributes=['first_name']).validate('testclient', user=user)
         )
 
+    def test_validate_property(self):
```

Comment:
When declaring models in test methods, you need to use [`@isolate_apps()`](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/unit-tests/#django.test.utils.isolate_apps).